stellar/its-example.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { Contract, Address, nativeToScVal } = require('@stellar/stellar-sdk');
+const { Command } = require('commander');
+const { loadConfig, saveConfig } = require('../evm/utils');
+const { getWallet, broadcast, addBaseOptions, tokenToScVal, tokenMetadataToScVal } = require('./utils');
+const { addOptionsToCommands, getChainConfig } = require('../common');
+const { ethers } = require('hardhat');
+const {
+    utils: { arrayify, hexZeroPad },
+} = ethers;
+require('./cli-utils');
+
+async function deployToken(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const minter = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const [symbol, name, decimal, salt, initialSupply] = args;
+    const saltBytes32 = hexZeroPad(salt.startsWith('0x') ? salt : '0x' + salt, 32);
+
+    const operation = contract.call(
+        'deploy_interchain_token',
+        caller,
+        nativeToScVal(Buffer.from(arrayify(saltBytes32)), { type: 'bytes' }),
+        tokenMetadataToScVal(decimal, name, symbol),
+        nativeToScVal(initialSupply, { type: 'i128' }),
+        minter,
+    );
+
+    await broadcast(operation, wallet, chain, 'Deploy Token', options);
```

Comment:
```suggestion
    await broadcast(operation, wallet, chain, 'Deploy Interchain Token', options);
```
or `Interchain Token Deployed`